internal/oci/runtime_oci.go

```diff
@@ -557,17 +559,17 @@ func waitContainerStop(ctx context.Context, c *Container, timeout time.Duration,
 			case <-chControl:
 				return
 			default:
-				process, err := findprocess.FindProcess(c.state.Pid)
+				process, err := findprocess.FindProcess(c.Pid())
```

Comment:
We can get the pid once at the beginning instead of calling into c.Pid() that take a read lock. The pid isn't expected to change.